addons/options/README.md

```diff
@@ -44,6 +44,7 @@ setOptions({
   showSearchBox: false,
   downPanelInRight: false,
   sortStoriesByKind: false,
+  hierarchySeparator: /\//,
```

Comment:
@joscha , currently `hierarchySeparator` is a string value, that represents regex, but not the RegExp object (because when serializing RegExp it becomes empty object, and everything in `setOptions` should be serializable) 

@UsulPro has a nice solution for that here -
 https://github.com/storybooks/storybook/pull/1394/files#r126082393